AutomationCucumber/src/test/java/features/CreateAccount.feature

```diff
@@ -12,15 +12,24 @@ Feature: Create Account
       | email        |  | message                |
       | invalidEmail |  | Invalid email address. |
 
+  Scenario Outline: Create Account Valid
+    Given User is on AutomationPractice page
+    When User click sing in.
+    Then User submits valid <Email>
+    Then User log in create an account page.
+
+    Examples:
+      | Email                        |
+      | roberto.bertrand@hotmail.com |
+
 
   Scenario Outline: Create An Account
     Given User is on AutomationPractice page
     When User click sing in.
-    Then User submits valid <Email>
     Then User log in create an account page.
```

Comment:
El usuario nunca hace log in. lo escrito en el step debe reflejar las acciones exactas que realiza el usuario.